praw/models/reddit/subreddit.py

```diff
@@ -17,6 +17,117 @@ class Subreddit(RedditBase, MessageableMixin, SubredditListingMixin):
     STR_FIELD = 'display_name'
     MESSAGE_PREFIX = '#'
 
+    @staticmethod
+    def _create_or_update(_reddit, allow_images=None, allow_top=None,
```

Comment:
In the event that reddit adds new settings, can we permit this to take **other_settings? That way the client can manually add them in their call and things _should_ continue to work.